documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -58,214 +54,116 @@ from additional images.
+and there are no files in the specified directory in the auxiliary image,
+then the domain deployment will fail.
 
-#### Model in Image paths
 
-For the Model In Image auxiliary image use case, you also need to
-configure the `domain.spec.configuration.model.modelHome`
-and `domain.spec.configuration.model.wdtInstallHome` attributes
-to specify the location of the domain's WebLogic Deploy Tool (WDT)
-model files and the domain's WDT installation.
-These default to `/u01/wdt/models` and `/u01/wdt/weblogic-deploy`
-respectively, and must be changed to specify a directory in
-`domain.spec.auxiliaryImageVolumes.mountPath`. For example:
+The files in `sourceModelHome` and `sourceWDTInstallHome` directories will be made available in `/aux/models` 
+and `/aux/weblogic-deploy` directories of the WebLogic Server container in all pods, respectively.
 
-```
-  configuration:
-    model:
-      modelHome: "/auxiliary/models"
-      wdtInstallHome: "/auxiliary/weblogic-deploy"
-```
+See [Configuration Example 2: Source Locations](#configuration-example-2-source-locations).
 
+#### Multiple Auxiliary Images
+If specifying multiple auxiliary images with model files in their respective `configuration.model.auxiliaryImages[].sourceModelHome`
+directories, then model files are merged. 
+The operator will merge the model files from multiple auxiliary images in the same order in which images appear under `model.auxiliaryImages`. 
+Files from later images in the merge overwrite same-named files from earlier images.
+
+When specifying multiple auxiliary images, ensure that only one of the images supplies a WDT installation location using 
+`configuration.model.auxiliaryImages[].sourceWDTInstallHome`. 
 {{% notice warning %}}
-If multiple auxiliary images supply different versions of a WebLogic Deploy Tool installation
-to the same `wdtInstallHome` path, then it is recommended 
-to ensure that the newer version completely replaces the older version
-instead of merges with it. See [Performing replaces instead of merges](#performing-replaces-instead-of-merges).
+If you provide more than one WDT install home using `sourceWDTInstallHome`, then the domain deployment will fail.
 {{% /notice %}}
 
-### Merge order
-
-Refer to this section if you need to control the merge order of files from 
-multiple auxiliary images that all reference the same volume, or if 
-you want to customize the command that is used to copy the
-files from an auxiliary image into its volume. You can
-use command customization to force "replace" behavior instead
-of merge behavior.
-
-#### Expected merge order
-
-By default, the files from multiple auxiliary images that share
-the same volume are merged. Specifically:
-
-- Files from later images in the merge overwrite same named files from earlier images.
-
-- The contents of overlapping directories from multiple images are combined.
+See [Configuration Example 3: Multiple Images](#configuration-example-3-multiple-images).
```

Comment:
See [Configuration Example 3: Multiple Images](#configuration-example-3-multiple-images). -> For an example of configuring multiple auxiliary images, see [Configuration example 3](#configuration-example-3-multiple-images).